editor/block-mover/test/mover-label.js

```diff
@@ -23,9 +18,9 @@ describe( 'block mover', () => {
 				true,
 				false,
 				dirUp,
-			) ).to.equal(
-				`Block "${ type }" is at the beginning of the content and can’t be moved up`
-			);
+			) ).toBe(
+                `Block "${ type }" is at the beginning of the content and can’t be moved up`
+            );
```

Comment:
Both lines 22 & 23 here are indented with spaces and not tabs

As to why ESLint is missing that I've no idea /shrug